lib/main/atom/commands/goToDeclaration.ts

```diff
@@ -27,12 +38,19 @@ commands.set("typescript:go-to-declaration", deps => {
     } else {
       open(result.body![0])
     }
-
-    function open(item: {file: string, start: {line: number, offset: number}}) {
-      atom.workspace.open(item.file, {
-        initialLine: item.start.line - 1,
-        initialColumn: item.start.offset - 1
-      })
-    }
   }
-})
+});
+
+commands.set("typescript:return-from-declaration", deps => {
+   return async e => {
+      const position = prevCursorPositions.pop();
+      if (!position) {
+         atom.notifications.addInfo('AtomTS: Previous position not found.');
+         return;
+      }
+      open({
+         file: position.file,
+         start: { line: position.line, offset: position.offset }
+      });
```

Comment:
The new object you're creating here has the exact same shape as `position` local, so there's no need for that. You could pass it directly, though you'll see that it's not what `open` expects.